java/pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>io.ceresdb</groupId>
     <artifactId>ceresdb-proto-internal</artifactId>
-    <version>0.1.0</version>
+    <version>0.3.0</version>
```

Comment:
We do not have 0.2.0. How about use 0.2.0 instead of 0.3.0

Reply:
i make a mistake,  fix already